articles/quickstart/spa/angular-beta/02-calling-an-api.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Calling an API
+description: This tutorial demonstrates how to call a backend API using an access token.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - angular
+  - login
+github:
+    path: 02-Calling-an-API
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD006 MD002 MD041 -->
+
+Most single-page apps use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](/api-auth).
+
+This tutorial shows you how to create a simple API using [Express](https://expressjs.com) that validates incoming JSON Web Tokens. You will then see how to call this API using an Access Token granted by the Auth0 authorization server.
```

Comment:
> ... that validates incoming JSON Web Tokens.

I know that's the primary task of that API but could you phrase this in a more general way? The API someone else is building would do that but not as a primary task. Consider:

> ... that serves resources protected by a middleware that looks for and validates access tokens.

... or something similar.

Reply:
@joshcanhelp Good shout. I think the text you've provided hits the nail on the head.